### ISSUE TITLE
Mouse clicks on UI act as clicks in game too

### ISSUE THREAD

**Issue by michael-house** [tsunamayo/Skywanderers#565]
1. Enter a ship or some other vehicle that responds to mouse clicks
2. Open the main menu
3. Click on menu items
4. Observe ship or vehicle responding to mouse clicks.

**Comment by tsunamayo** [comment]
Hi, okay I cant reproduce this one.
Did you used a starting ship? If yes which one?
Is the mouse button coming from an hotkey or a mapping from your configuration?
Thanks!

**Comment by michael-house** [comment]
Yes, this is inside one of the starting space ships, it has two fire modes bound to right and left click. The keybindings are set to default. I can try to get a screen capture if you're still unable to reproduce.
